faq/account.mdx

```diff
@@ -0,0 +1,144 @@
+---
+meta:
+  title: Account
+  description: Get answers to your most common account-related questions.
+content:
+  h1: Account
+hero: assets/general.webp
+---
+
+<Concept>
+
+## My account is locked. What do I do?
+
+If your account is locked, [contact our support team](https://console.scaleway.com/support/tickets/create) to learn the reason for the blockage and discover how to regain access.
+
+</Concept>
+
+<Concept>
+
+## Can I change my Organization’s country?
+
+If you need to change your Organization country and region, you must [create a support ticket](https://console.scaleway.com/support/tickets/create).
+
+To make the process faster, when creating your ticket, inform us of your Organization’s new address, including: street address, city, postal code, region, and country.
+
+</Concept>
+
+<Concept>
+
+## Can I move a Project to another account?
+
+At this moment, it is not possible to move a Project from one account to another.
+
+</Concept>
+
+<Concept>
+
+## Can I recover a deleted account?
+
+No, it is not possible to recover a deleted account.
+
+If you only want to pause your account and keep your existing resources, you can [deactivate it](/console/account/how-to/close-account/#how-to-deactivate-your-organization) instead.
+
+</Concept>
+
+<Concept>
+
+## How do I change my email address?
+
+To change your email address:
+
+1. On the [Scaleway console](https://console.scaleway.com), go to your [profile](https://console.scaleway.com/account/profile).
+2. Under the section **User information**, click **Edit profile**.
+3. Enter a new address under **Email address**, then click **Confirm changes** to save your modification.
+
+Your email address is now updated.
```

Comment:
```suggestion
Confirm your new email address by clicking the link sent to your inbox. Your email address is now updated.
```

Reply:
Thanks! Added the first part as a 4th step. :)